hackathon/templates/hackathon/hackathon_list.html

```diff
@@ -11,9 +11,20 @@ <h2 class="p-orange text-center mb-4">Hackathons</h2>
     {% else %}
         <h2 class="text-center">There aren't any Hackathons at the moment!</h2>
     {% endif %}
+
+    <!-- Create event button only available for admin users -->
```

Comment:
Not really sure this is the best place for the button. It would be at the bottom of the page and I guess you would really want it at the top where we already have it in the navbar. So I would just remove this unless there is any specific use case for it?

Reply:
True that, there isn't a specific use case for it, I just thought I would include it in there too and see what you thought. I do agree with you though, so I've removed it from the template. This will be visible when I push the new changes to the existing PR.